packages/stacks-classic/lib/components/empty-state/empty-state.less

```
@@ -1,15 +1,22 @@
 .s-empty-state {
     // CHILD ELEMENTS
     p {
-        strong {
-            color: var(--fc-dark);
-        }
-
+        color: var(--black-400);
```

In cases where there are no additional elements after the `p`, there's a residual 32px of padding below the `p` element.

We can have this padding only apply when there's something after the `p` using the `last-child` selector:
```suggestion
        &:last-child {
            margin-bottom: 0;
        }

        color: var(--black-400);
```

---

That's really clever! Thanks, made the change.